src/common/schedulers/slurm_commands.py

```diff
@@ -149,6 +147,26 @@ def __repr__(self):
     def __str__(self):
         return f"{self.name}({self.nodeaddr})"
 
+    def is_reboot_requested(self):
+        return self.SLURM_SCONTROL_REBOOT_REQUESTED_STATE in self.states
+
+    def is_reboot_issued(self):
+        return self.SLURM_SCONTROL_REBOOT_ISSUED_STATE in self.states
+
+    def is_rebooting(self):
+        """
+        Check if the node is rebooting via scontrol reboot.
+        Check that the node is in a state consistent with the scontrol reboot request.
```

Comment:
Missing a blank line between line 158 and 159:
see code-check results for redetails:
```
${cyan}src/common/schedulers/slurm_commands.py${reset}:${yellow_bold}157${reset}:${green_bold}1${reset}: ${red_bold}D205${reset} 1 blank line required between summary line and description
```